src/fields/valueField.ts

```diff
@@ -123,9 +124,19 @@ export function newValueFieldState<T, K extends keyof T>(
       return parentState().readOnly || this._readOnly;
     },
 
-    /** Sets the field readOnly, but `v` will still be or-d with the parent's readOnly state. */
-    set readOnly(v: boolean) {
-      this._readOnly = v;
+    /** Sets the field readOnly, but `loading` will still be or-d with the parent's readOnly state. */
+    set readOnly(loading: boolean) {
```

Comment:
I kind of find it odd that the parameter's name is `loading`. Seems to couple the readOnly state with the loading state, where I'd think `readOnly` can be `true`, even if `loading === false`.

Reply:
was this on purpose?